lib/request.js

```diff
@@ -64,11 +64,15 @@ class REQUEST {
     this.query = this.app._event.queryStringParameters ? this.app._event.queryStringParameters : {}
 
     // Set the raw headers
-    this.rawHeaders = this.app._event.headers || {}
+    this.rawHeaders = Object.assign({}, this.app._event.headers, this.app._event.multiValueHeaders)
```

Comment:
ALB without 'multi-value header' support turned on doesn't contain this value. I'll add a fallback.

Reply:
the good thing of Object assign is that if your are trying to assign something that is `undefined` that property will be ignored
` Object.assing({test: 'hello'}, {a: 2}, undefined)` is `{test: 'hello', a: 2}`

Reply:
Sorry, didn't see that you had the regular `headers` in there. This will produce a mix of strings and arrays. I want to avoid that.

Reply:
```suggestion
    const headers = Object.assign({}, this.app._event.headers, this.app._event.multiValueHeaders)
    this.rawHeaders = Object.keys(headers).reduce((acc, e) => Object.assign(acc, { [e]: headers[e] instanceof Array ? headers[e] : [headers[e]] }), {})
```